lab2/lab2-Symbol Table/src/SymbolTable.java

```diff
@@ -1,63 +1,64 @@
-import java.util.AbstractMap;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Map;
+//import java.util.AbstractMap;
+//import java.util.ArrayList;
+//import java.util.List;
+//import java.util.Map;
+//
+//public class SymbolTable {
+//
+//    private List<List<String>> hashTable;
+//
+//    // Current capacity of hashes list
+//    private int numBuckets;
+//
+//    // Current size of array list
+//    private int size;
+//
+//    // Constructor (Initializes capacity, size and
+//    // empty chains.
+//    public SymbolTable()
+//    {
+//        hashTable = new ArrayList<>();
+//        numBuckets = 257;
```

Comment:
i think numBuckets should have the value of 256, because that's how many characters the ASCII code has